.mocharc.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
We this config file impact the current mocha tests?

Reply:
Maybe to work around this, we can think of changing the name from default to some customized one, to avoid other test command to read it, and then specify it in the e2e like `-f config.json`